charts/cluster-autoscaler/Chart.yaml

```diff
@@ -11,4 +11,4 @@ name: cluster-autoscaler
 sources:
   - https://github.com/kubernetes/autoscaler/tree/master/cluster-autoscaler
 type: application
-version: 9.35.0
+version: 9.36.0
```

Comment:
```suggestion
version: 9.35.1
```

Reply:
It should be the patch version, not the minor version.

Reply:
Disagree with this, we're adding new functionality, this should be a minor version bump.

Reply:
Hi @gjtempleton, IIRC previously while adding support for exoscale provider, we updated the patch version.
Here is the PR: #6249, so I did the above review. But if you think it is good to have a minor version then I have no problem with it.
Thanks for the above information.

Reply:
@NiklasRosenstein could you please revert the version to `9.36.0`, as suggested by @gjtempleton.

Reply:
Done!